cloud-agent/install.sh

```
@@ -0,0 +1,90 @@
+  fi
+
+  if command -v systemctl >/dev/null 2>&1 && [ -d /run/systemd/system ]; then
+    if sudo systemctl enable --now docker 2>/dev/null; then
+      return 0
+    fi
+    echo "systemctl could not start docker; falling back to dockerd."
+  fi
+
+  if ! pgrep -x dockerd >/dev/null 2>&1; then
+    echo "Starting dockerd in the background..."
+    sudo dockerd >/tmp/dockerd.log 2>&1 &
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)